packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewerConnector.js

```diff
@@ -0,0 +1,32 @@
+import { withStores } from '@helpers'
+import ImageAndTextViewerContainer from './ImageAndTextViewerContainer'
+
+function storeMapper (store) {
+  const {
+    subjects: {
+      active: subject
```

Comment:
This is passed down by the `SubjectViewer` component, so I think you can leave it out here.